src/test/msixtest/api_manifestreader.cpp

```diff
@@ -623,13 +623,13 @@ TEST_CASE("Api_AppxManifestReader_OptionalPackage", "[api]")
 
     std::vector<std::string> expectedPublisherValues =
     {
-        "", // We expect the value to be NULL. Handle this special case separately  
+        "CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US",  

```

Comment:
>"CN=Microsoft Corporation, O=Microsoft Corporation, L=Redmond, S=Washington, C=US", [](start = 8, length = 83)

Just to be sure, you are saying that the previous implementation was wrong and that this one matches the windows implementation?

Reply:
Correct. My original implementation did not match the windows implementation, and this change fixes that.